test/bootswatch4_test.js

```diff
@@ -39,6 +39,29 @@ describe('bootswatch4', () => {
         helpers.assert.bodyClass('page-bootswatch', response, done);
     });
 
+    const totalThemes = config.bootswatch4.themes.length;
+    const invalidQueries = [
+        parseInt(-1, 10),
+        parseInt(0, 10),
+        parseInt(1, 10),
+        parseInt(500, 10),
+        parseInt('5', 10),
+        parseInt(totalThemes - 1, 10),
+        parseInt('foobar', 10)
```

Comment:
I would expect 0, 1 and Len-1 to return valid items in the array. They don’t?

Reply:
Yeah, they do. This just tests that an error is not thrown. Maybe we should tweak the test more to make it clear.

Reply:
I'll leave that to you. Currently it reads that all those values are invalid though.

Reply:
True, I removed those for now.